browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -121,7 +121,7 @@ base::WeakPtr<SpeedreaderTabHelper> SpeedreaderTabHelper::GetWeakPtr() {
 
 bool SpeedreaderTabHelper::IsSpeedreaderEnabled() const {
 #if BUILDFLAG(IS_ANDROID)
-  return true;  // skip the preference storage for now
+  return true;  // TODO(Wengling): skip the preference storage for now
```

Comment:
let's add a Preference to control that as well, place it near the simplified mode view (Settings->Appearance). Call it same way as we have it on desktop. Let's make it hidden if flag is off and it is off by default. When a speedreader selected let's hide the simplified page view control and only show it when speedreader is disabled.

Reply:
I have added the preference and also hide the simplified view setting.